src/components/profiles-modals/ImportProfileModal.vue

```diff
@@ -297,67 +314,46 @@ export default class ImportProfileModal extends mixins(ProfilesMixin) {
         </template>
     </ModalCard>
 
+    <ModalCard v-else-if="activeStep === 'REFRESH_MOD_LIST'" key="REFRESH_MOD_LIST" :is-active="isOpen" :can-close="false">
+        <template v-slot:header>
+            <h2 class="modal-title">Refreshing mod list</h2>
+        </template>
+        <template v-slot:footer>
+            <div>
+                <p>
+                    At least some of the packages were not found. We're trying to fix this by refreshing the online mod list:</br>
```

Comment:
Minor: I wonder if it's clear to the users what "not found" means in this context? Would something like this be clearer?

> Some of the packages in the profile are not recognized by the mod manager. Refreshing the online mod list might fix the problem.

Reply:
Yeah, much better, I'll do this 👍